### DISCUSSION TITLE
new command /cmi whitelistlist

### DISCUSSION THREAD

**Discussion by LivingDarkness** [Zrips/CMI#7723] [Feature Requests]
add the /cmi whitelistlist command, similar to the /cmi oplist command, so you can see the list of people in the whitelist one below the other.

**Comment by CloudeLecaw** [comment]
you mean the whitelist of whitelisted servers?   hmm why not?
but to be honest this allready exist by vanilla...
Do so we need to invent a new wheel when its already existing?

> Commands   - Description
> whitelist add <targets> (Java Edition)   - Adds the player profile(s) to the whitelist. The player does not need to be online.
> whitelist add <name: string> (Bedrock Edition) 
> whitelist list  - Displays all player profiles in the whitelist.
> whitelist off  - Disables the server's use of a whitelist.
> whitelist on  - Enables the server's use of a whitelist.
> whitelist reload  - Reloads the list of player profiles in whitelist.json from disk (used when whitelist.json has been modified outside of Minecraft).
> whitelist remove <targets> (Java Edition)   - Removes the player profile(s) from the whitelist. The player does not need to be online.
> whitelist remove <name: string> (Bedrock Edition)

<html>
<body>


Command | Description | Command Example | Permission
-- | -- | -- | --
/whitelist on/off, /whitelist add/remove player, /whitelist list, /whitelist reload | Enable or disable whitelisting (i.e. only listed players may join), Add or remove player from the whitelist, Lists whitelist, Reloads whitelist from file. | /whitelist on, /whitelist add player1, /whitelist list, /whitelist reload | minecraft.command.whitelist




</body>
</html>

https://minecraft.fandom.com/wiki/Commands/whitelist
https://bukkit.fandom.com/wiki/CraftBukkit_Commands

**Comment by LivingDarkness** [comment]
I did not intend to add the Whitelist commands, I wanted the list of players in the Whitelist to be managed one under the other as in the /oplist command, to make everything more ordered.

**Comment by CloudeLecaw** [comment]
/whitelist list ?

Isnt that order there enough?

**Comment by LivingDarkness** [comment]
in my opinion it is not ordered, see attached photo. as you can see the first message is from /whitelist list and the second from /oplist, the op people list is much more sorted, can't make this function similar for whitelist?

![Screenshot_6](https://user-images.githubusercontent.com/83078827/151981911-0261d454-f3fd-4fae-b744-d66c85d98bf5.png)

**Comment by CloudeLecaw** [comment]
Pls move it to discussions :)
